jena-fuseki-kafka-module/pom.xml

```diff
@@ -92,15 +92,11 @@
       </exclusions>
     </dependency>
 
-
-    <!-- Temporary dependency -->
-    <!-- CVE-2024-7254 -->
     <dependency>
-      <groupId>com.google.protobuf</groupId>
-      <artifactId>protobuf-java</artifactId>
-      <version>${dependency.protobuf}</version>
+      <groupId>commons-io</groupId>
```

Comment:
What's with this dependency? Another CVE?

Reply:
Supposed to be test scope as an older version was getting pulled in which was breaking Test Containers